File: src/main/java/net/dashflight/data/jwt/verify/DashflightJwtVerifierModule.java
package net.dashflight.data.jwt.verify;

import com.google.inject.AbstractModule;
import net.dashflight.data.jwt.verify.request.DashflightVerifyJwtRequestProviderModule;

public class DashflightJwtVerifierModule extends AbstractModule {

    @Override
    protected void configure() {
        install(new DashflightVerifyJwtRequestProviderModule());

        bind(JwtVerifier.class);
    }

}

File: src/main/java/net/dashflight/data/jwt/verify/DashflightJwtVerifierModule.kt
package net.dashflight.data.jwt.verify

import com.google.inject.AbstractModule
import net.dashflight.data.jwt.verify.request.DashflightVerifyJwtRequestProviderModule

class DashflightJwtVerifierModule : AbstractModule() {
    override fun configure() {
        install(DashflightVerifyJwtRequestProviderModule())
        bind(JwtVerifier::class.java)
    }
}